src/SourceBuild/patches/roslyn/0001-Use-Content-for-non-TFM-specific-items.patch

```diff
@@ -0,0 +1,35 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Nikola Milosavljevic <nikolam@microsoft.com>
+Date: Thu, 30 Mar 2023 22:30:25 +0000
+Subject: [PATCH] Use Content for non-TFM specific items
+
```

Comment:
Please include a "Backport: <PR/Issue>" comment on all patches.  This is very helpful for tracking purposes.